src/routes/docs/quick-starts/kotlin/+page.markdoc

```diff
@@ -0,0 +1,173 @@
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+Then, under **Integrate with your server**, add an **API Key** with the following scopes.
+
+{% only_dark %}
+![Server integrations](/images/docs/quick-starts/dark/integrate-server.png)
+{% /only_dark %}
+{% only_light %}
+![Server integrations](/images/docs/quick-starts/integrate-server.png)
+{% /only_light %}
+| Category  {% width=120 %} | Required scopes       | Purpose |
+|-----------|-----------------------|---------|
+| Database  | `databases.write`     | Allows API key to create, update, and delete [databases](/docs/products/databases/databases). |
+|           | `collections.write`   | Allows API key to create, update, and delete [collections](/docs/products/databases/collections). |
+|           | `attributes.write`    | Allows API key to create, update, and delete [attributes](/docs/products/databases/collections#attributes). |
+|           | `documents.read`      | Allows API key to create, update, and delete [documents](/docs/products/databases/documents). |
+|           | `documents.write`     | Allows API key to read [documents](/docs/products/databases/documents). |
+
+Other scopes are optional.
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+{% /section %}
+{% section #step-2 step=2 title="Create Kotlin project" %}
+Create a Kotlin application by opening **IntelliJ IDEA** > **New Project** and create a **Kotlin** application.
+This quick start will use **Gradle** as the build system, but you can follow with Maven or IntelliJ if you're more comfortable.
+
+Follow the wizard and open your new project. 
+
+{% /section %}
+{% section #step-3 step=3 title="Install Appwrite" %}
+Open you `build.gradle.kts` file and implement the following dependency.
+
+```groovy
+dependencies {
+    ... other dependencies
+    implementation("io.appwrite:sdk-for-kotlin:4.1.0")
+}
+```
+
+{% /section %}
+{% section #step-4 step=4 title="Import Appwrite" %}
+
+Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier. 
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+Open the file `Main.kt` and initialize the Appwrite Client. Replace `<YOUR_PROJECT_ID>` with your project ID and `<YOUR_API_KEY>` with your API key.
+
+```kotlin
+import io.appwrite.Client
+import io.appwrite.services.Databases
+import io.appwrite.models.Database as DatabaseModel
```

Comment:
Why are we importing as `DatabaseModel` as service is `Databases` there shouldn't be conflict?

Reply:
Fixed